### ISSUE TITLE
invalid usage of tag >

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#1141]
when compiling with java 11 you will get following javadoc warnings

```
/data/xtext-master/git/xtext-core/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/ITextRegionDiffBuilder.java:24: warning - Tag @link: reference not found: IHiddenRegionPartMerger
/data/xtext-master/git/xtext-core/org.eclipse.xtext/src-gen/org/eclipse/xtext/serializer/XtextSemanticSequencer.java:451: warning - invalid usage of tag >
/data/xtext-master/git/xtext-core/org.eclipse.xtext/src-gen/org/eclipse/xtext/serializer/XtextSemanticSequencer.java:451: warning - invalid usage of tag >
/data/xtext-master/git/xtext-core/org.eclipse.xtext/src-gen/org/eclipse/xtext/serializer/XtextSemanticSequencer.java:482: warning - invalid usage of tag >
/data/xtext-master/git/xtext-core/org.eclipse.xtext/src-gen/org/eclipse/xtext/serializer/XtextSemanticSequencer.java:482: warning - invalid usage of tag >

```

```
/**
	 * Contexts:
	 *     Alternatives returns Group
	 *     Alternatives.Alternatives_1_0 returns Group
	 *     ConditionalBranch returns Group
	 *     UnorderedGroup returns Group
	 *     UnorderedGroup.UnorderedGroup_1_0 returns Group
	 *     Group returns Group
	 *     Group.Group_1_0 returns Group
	 *     AbstractToken returns Group
	 *     AbstractTokenWithCardinality returns Group
	 *     AbstractTerminal returns Group
	 *     ParenthesizedElement returns Group
	 *
	 * Constraint:
	 *     (
	 *         (
	 *             (guardCondition=Disjunction elements+=AbstractToken+) | 
	 *             (elements+=Group_Group_1_0 elements+=AbstractToken+) | 
	 *             ((predicated?='=>' | firstSetPredicated?='->') elements+=Alternatives)
	 *         ) 
	 *         (cardinality='?' | cardinality='*' | cardinality='+')*
	 *     )
	 */
```

=> looks like the escaping misses > <

**Comment by cdietrich** [comment]
https://reflectoring.io/howto-format-code-snippets-in-javadoc/

**Comment by cdietrich** [comment]
another place is

```
Ambiguous syntax:
	 *     '('*
	 *
	 * This ambiguous syntax occurs at:
	 *     (rule start) (ambiguity) '<' guardCondition=Disjunction
	 *     (rule start) (ambiguity) '{' type=TypeRef
	 *     (rule start) (ambiguity) feature=ValidID
	 *     (rule start) (ambiguity) firstSetPredicated?='->'
	 *     (rule start) (ambiguity) predicated?='=>'
	 *     (rule start) (ambiguity) rule=[AbstractRule|RuleID]
	 *     (rule start) (ambiguity) value=STRING
	 *     (rule start) (ambiguity) {Alternatives.elements+=}
	 *     (rule start) (ambiguity) {Group.elements+=}
	 *     (rule start) (ambiguity) {UnorderedGroup.elements+=}
```

**Comment by cdietrich** [comment]
there `@` might be a symbol too

**Comment by cdietrich** [comment]
=> no easy/quick solution

**Comment by cdietrich** [comment]
Fixed in 2.26
